code/modules/library/lib_items.dm

```diff
@@ -206,6 +206,9 @@
 	var/runestun = 0	//Does it have a stun talisman in it?
 	var/occult = 0 //Does this book contain forbidden and occult writings?
 
+	var/book_width = 400
+	var/book_height = 400
```

Comment:
I wonder if it would be worthwhile to simply move display_x and display_y (from paper.dm) up to a higher level.

Reply:
considering most items in the game are weapons but don't make use of those I'm not sure.